combat-trainer.lic

```diff
@@ -607,20 +610,21 @@ class SpellProcess
   def check_charging?(game_state)
     return false unless game_state.charge_cambrinth?
 
-    find_cambrinth3(@cambrinth, @held_cambrinth)
```

Comment:
I don't recall doing anything with some of these lines... is this a product of merging?